proposed/2021/NuGetOrgDetailsPageRedesign.md

```diff
@@ -0,0 +1,140 @@
+
+![](../../meta/resources/NuGetPackageDetails/overallnew.png)
+
+### Version 1: Tabs over Sidebar
+
+![](../../meta/resources/NuGetPackageDetails/overallreal.png)
+
+### Version 2: Tabs next to Sidebar
+
+![](../../meta/resources/NuGetPackageDetails/overallv2.png)
+## Motivation
+
+We are running out of space on the page. We have lots of features we want to add like READMEs and changelogs. Currently, those additions would push content down the screen so much that users can’t read it. 
+
+### Problem: What problem is this solving?
+
+It is difficult for developers to use NuGet.org for finding packages. They struggle to navigate the site and find the info they need. They use external documentation on GitHub and personal blogs instead of the package details page.
+
+### Why: How do we know this is a real problem and worth solving?
+
+We have received many requests for UI improvements on GitHub. Developers have mentioned consistently in customer interviews that they do not like to use NuGet.org for finding and validating  packages and will look for other resources whenever they can. The information is not organized intuitively and caused users to scroll to the bottom of the page or use extra clicks. In addition, about 20% of our traffic comes from mobile usage and we need to make sure our design continues to work with mobile.   
+
+We cannot improve the NuGet.org package details page entirely with small, undirected changes. Instead we need a coherent, comprehensive, and most importantly, modular plan of action that addresses the core problems but can be easily separated into manageable chunks of work.
+
+
+### Success: How do we know if we’ve solved this problem?
+- By HaTS survey results and customer interview replies  
+- By measuring rates at which people click on sections/tabs on the package details page  
+- By measuring changes in the NuGet.org traffic for the details page of our top 100 popular packages
+- By gauging excitement/disappointment after blogging about this feature & analyzing sentiment on Twitter/GitHub/DevCom/etc.
+
+
+### Audience: Who are we building for?
+
+We are building this for the .NET developer who is browsing & making decisions about including a NuGet package. They want to:
+-	Find information they need quickly and easily
+-	Use higher quality packages
+-	Use more secure packages
+
+We are also building this for the .NET developer who is creating & publishing NuGet packages for public use. They want to:
+-	Display relevant information about their package
+-	Increase usage of their package
+
+## Goals and Non-Goals
+
+### Header
+
+![](../../meta/resources/NuGetPackageDetails/headernew.png)
+
+Information that needs to be seen at a glance such as package ID, description, and eventually, TFM badges.
+
+**Requirements**        
+- Title   
+- Icon  
+    - Hidden for packages without icon
+- Version   
```

Comment:
Consider showing when the version being viewed was published. Based on feedback in the issue and customer interviews, how "updated" a package is is a critical detail. It may also be worthwhile to surface the last time to package was updated as a whole in the sidebar.

Reply:
Love it! Will make the edit